### ISSUE TITLE
poor prediction in satellite images

### ISSUE THREAD

**Issue by jaigsingla** [mapbox/robosat#219]
Hi All,

I am able to get 0.83 of IOU using the Tanzania dataset using your code and able to generate vector files after predictions.  
Now, I have a 50cm VHR image over an urban area and corresponding building shape files in geojson format. I applied the same techniques as in the Tanzania dataset and trained the model with an IOU of .67
But, generated segmentation probabilities do not look good, what can be the reason ? 
I have used batch_size of 16, image_size of 512, lr= 0.00005 and loss function of Lovasz.
I will be happy to hear from you.

regards,
Jai

**Comment by daniel-j-h** [comment]
Can be anything from
- not enough data
- poor optimizer settings (lr too low?)
- not trained for long enough
- trained for too long
- ..

By now this project is outdated, unmaintained, and no longer state of the art, too.

**Comment by jaigsingla** [comment]
Hi Daniel,

You are right it may be anything. Can you guide me through? It will be
preety useful. In case you have commercial software, pls let me know.

**Comment by manapshymyr-OB** [comment]
> Hi All,
> 
> I am able to get 0.83 of IOU using the Tanzania dataset using your code and able to generate vector files after predictions.
> Now, I have a 50cm VHR image over an urban area and corresponding building shape files in geojson format. I applied the same techniques as in the Tanzania dataset and trained the model with an IOU of .67
> But, generated segmentation probabilities do not look good, what can be the reason ?
> I have used batch_size of 16, image_size of 512, lr= 0.00005 and loss function of Lovasz.
> I will be happy to hear from you.
> 
> regards,
> Jai

Hello! 
on which OS have you performed this project?

**Comment by manapshymyr-OB** [comment]
> Can be anything from - not enough data - poor optimizer settings (lr too low?) - not trained for long enough - trained for too long - .. By now this project is outdated, unmaintained, and no longer state of the art, too.
> […](#)
> On June 7, 2021 9:07:17 AM UTC, jaigsingla ***@***.***> wrote: Hi All, I am able to get 0.83 of IOU using the Tanzania dataset using your code and able to generate vector files after predictions. Now, I have a 50cm VHR image over an urban area and corresponding building shape files in geojson format. I applied the same techniques as in the Tanzania dataset and trained the model with an IOU of .67 But, generated segmentation probabilities do not look good, what can be the reason ? I have used batch_size of 16, image_size of 512, lr= 0.00005 and loss function of Lovasz. I will be happy to hear from you. regards, Jai

Hello!
Is it possible to run on OS Windows without Docker?
